### PR TITLE
Set the env vars and labels for the s2i-minimal images

### PR DESCRIPTION
**Pull request by harshad16**
Set the env vars and labels for the s2i-minimal images
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: https://github.com/thoth-station/thoth-application/issues/1788

### CODE REVIEW

File: overlays/python36/Dockerfile [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
-      io.openshift.expose-services="8888:http" \
-      io.openshift.tags="python,python36" \
-      authoritative-source-url="https://quay.io/thoth-station/s2i-minimal-notebook" \
-      io.openshift.s2i.build.commit.ref="master" \
-      io.openshift.s2i.build.source-location="https://github/thoth-station/s2i-minimal-notebook" \
-      io.openshift.s2i.build.image="quay.io/thoth-station/s2i-thoth-ubi8-py36:v0.32.0"
 
-ENV JUPYTER_ENABLE_LAB="1" \
+ARG IMAGE_NAME="quay.io/thoth-station/s2i-minimal-notebook"
+ARG IMAGE_TAG="latest"
```
Comment (Gregory-Pereira) [inline overlays/python36/Dockerfile:4]: Why are the UBI images done this way with an arguement instead of passing the variable through?

**Review by Gregory-Pereira — COMMENTED** [review]
/lgtm

File: overlays/python36/Dockerfile (hunk 1 above)
Comment (harshad16) [reply overlays/python36/Dockerfile]: Normally args are used for passing value during buildtime.
 removed it as we dont need it.

**Pull request merged**